src/components/Toast/README.md

```diff
@@ -236,7 +236,7 @@ class ToastExample extends React.Component {
       <Toast
         content="Message sent"
         onDismiss={this.toggleToast}
-        duration={4500}
+        duration={Duration.Fast}
```

Comment:
I think this needs to be `ToastDuration` - to match the name of the export in Polaris otherwise our examples might complain about an unknown variable

Reply:
Oh good grief. 🤦‍♀️ Thank you